test/Unit.purs

```diff
@@ -38,11 +42,15 @@ import Test.Ctl.Types.TokenName as Types.TokenName
 import Test.Ctl.Types.Transaction as Types.Transaction
 import Test.Ctl.UsedTxOuts as UsedTxOuts
 import Test.Ctl.Wallet.Cip30.SignData as Cip30SignData
+import Test.Spec.Runner (defaultConfig)
 
 -- Run with `spago test --main Test.Ctl.Unit`
 main :: Effect Unit
-main = launchAff_ do
-  interpret testPlan
+main = interruptOnSignal SIGINT =<< launchAff do
+  flip cancelWith (effectCanceler (exitCode 1)) do
+    interpretWithConfig
+      defaultConfig { timeout = Just $ Milliseconds 30_000.0, exit = true }
+      testPlan
```

Comment:
Probably not necessary, seeing as we don't launch plutip in unit/integration. However, it doesn't hurt to have this (enables graceful shutdown of Aff)